string_helpers.py

```diff
@@ -1,29 +1,41 @@
-###############################################################
-def string_used_words(used_letters): # Crea lista de letras usadas con espacio en el medio
+def show_used_letters(used_letters):
+    """
+    Converts list to str and concatenates ' '
+    """ 
     result = ""
     for x in used_letters:
         result += x.upper() + " "
     return result
```

Comment:
the python way

Hay algo que ya veremos mas adelante que se llama "comprehension lists", y basicamente es armar listas de este estilo en una sola linea. En este caso se podria hacer:

```
result = "".join([x.upper() + " " for x in used_letters])
```

`[x.upper() + " " for x in used letters]` te va a devolver la lista ["A ", "B ", "C "]
`"".join()` te pega toda esa lista con un caracter vacio